instrumentation/mybatis/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/mybatis/MybatisSpanNameExtractor.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.mybatis;
+
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+
+public class MybatisSpanNameExtractor implements SpanNameExtractor<MapperMethodRequest> {
+
+  @Override
+  public String extract(MapperMethodRequest request) {
+    return request.getMapperName();
```

Comment:
could also consider replacing this class with a lambda

Reply:
> could also consider replacing this class with a lambda

I adjusted it.